ta/sks/include/sks_ta.h

```diff
@@ -13,6 +13,10 @@
 #define TA_SKS_UUID { 0xfd02c9da, 0x306c, 0x48c7, \
 			{ 0xa4, 0x9c, 0xbb, 0xd8, 0x27, 0xae, 0x86, 0xee } }
 
+/* SKS trusted application version information */
+#define SKS_VERSION_ID0		0
```

Comment:
I doubt that this will be very useful.
I'd prefer using capabilities to describe what's available in the TA. We could have both too of course.
If you keep the version stuff, please use the major-minor version scheme.

Reply:
True that there is currently no specification of what this version info describes.

Reply:
Regarding the PKCS#11 token capabilities, there are already means in the PKCS#11 API (are related SKS TA commands) for a client to get token capabilities.

This version stuff is more a hint in case the TA API changes in some way. For example SKS TA currently targets PKCS#11 2.40-e01. This hint could help moving to a later revision. Capabilities are more flag based, no?

I will keep this version hint (renamed major/minor). We'll introduce a command for capabilities exchange once there a need for.